lib/pdc/v1/release_variant.rb

```diff
@@ -1,13 +1,18 @@
 module PDC::V1
   class ReleaseVariant < PDC::Base
-    self.primary_key = :id
 
     attributes :release, :uid, :name, :type, :arches
 
     def release
       Release.find(attributes[:release])
     end
 
+    def url
+      URI.join(PDC.config.site, PDC.config.api_root,
+               PDC::Resource::Path.new(self.class.resource_path + '/(:release)/(:uid)', attributes).expanded).to_s
```

Comment:
With the change proposed above, will the Base class implementation of `url` work?